ekip/everykid/templates/get-your-pass/student_pass.html

```diff
@@ -4,23 +4,25 @@
 
 {% block page_id %}student-pass{% endblock page_id %}
 
-{% block background %}brown-rocks{% endblock background %}
+{% block background %}tent{% endblock background %}
 
 {% block content %}
 
-    <h2>4th graders:</h2>
-    <h3>Start your outdoor adventure today.</h3>
+  <div class="pass-content">
 
-    <p>Play a game and discover all the wonderful places you can go. After the
```

Comment:
Change this text to:

Start your outdoor adventure right now

Play a game and discover all the wonderful places you can go. After the game, you can print a voucher for free access to your federal lands and waters. 

This voucher can then be exchanged for your free 4th grade pass (a more durable, and smaller version) at many federal lands and waters (don't worry, we'll tell you where).